packages/maskbook/src/plugins/FileService/SNSAdaptor/components/Uploading.tsx

```diff
@@ -1,13 +1,13 @@
 import { Grid, makeStyles, Typography } from '@material-ui/core'
-import { useState, useEffect } from 'react'
+import { useEffect, useState } from 'react'
 import { File } from 'react-feather'
 import { useHistory, useLocation } from 'react-router'
 import { useAsync } from 'react-use'
-import { useI18N, timeout } from '../../../utils'
-import { PluginFileServiceRPC, PluginFileServiceRPCGenerator } from '../utils'
-import { FileRouter } from '../constants'
+import { timeout, useI18N } from '../../../../utils'
+import { FileRouter } from '../../constants'
+import type { FileInfo } from '../../types'
+import { PluginFileServiceRPC, PluginFileServiceRPCGenerator } from '../../utils'
```

Comment:
Please add a `Worker` entry for this plugin and import this file, otherwise, the RPC won't respond because it isn't started in the background page.